dev/manifest.txt

```diff
@@ -0,0 +1,22 @@
+app
+bin
+config
+conjur
```

Comment:
I **think** we can omit `conjur` directory.
In fact, can this directory be removed from the repository? I don't see where it's used.

Reply:
I agree, I think we can delete it

Reply:
Deleted